webapp/package.json

```diff
@@ -13,62 +13,72 @@
+    "@babel/preset-typescript": "7.16.7",
+    "@babel/runtime": "7.16.7",
+    "@emotion/babel-preset-css-prop": "11.2.0",
+    "@emotion/core": "10.3.1",
+    "@mattermost/types": "6.7.0-0",
+    "@testing-library/jest-dom": "5.16.1",
+    "@types/babel__core": "7.1.18",
+    "@types/babel__template": "7.4.1",
+    "@types/enzyme": "3.10.11",
+    "@types/jest": "27.4.0",
+    "@types/node": "17.0.12",
+    "@types/react": "16.14.26",
+    "@types/react-dom": "17.0.11",
+    "@types/react-redux": "7.1.22",
     "@types/react-router-dom": "5.1.5",
     "@types/react-transition-group": "4.4.0",
-    "@typescript-eslint/eslint-plugin": "3.6.0",
-    "@typescript-eslint/parser": "3.6.0",
+    "@typescript-eslint/eslint-plugin": "5.10.1",
+    "@typescript-eslint/parser": "5.10.1",
     "babel-eslint": "10.1.0",
-    "babel-jest": "26.1.0",
-    "babel-loader": "8.1.0",
-    "babel-plugin-typescript-to-proptypes": "1.3.2",
-    "css-loader": "3.6.0",
+    "babel-loader": "8.2.2",
+    "babel-plugin-formatjs": "10.3.7",
+    "babel-plugin-typescript-to-proptypes": "2.0.0",
+    "css-loader": "6.5.1",
     "enzyme": "3.11.0",
-    "enzyme-adapter-react-16": "1.15.2",
-    "enzyme-to-json": "3.5.0",
-    "eslint": "7.4.0",
-    "eslint-import-resolver-webpack": "0.12.2",
-    "eslint-plugin-import": "2.22.0",
-    "eslint-plugin-react": "7.20.3",
-    "eslint-plugin-react-hooks": "4.0.6",
-    "file-loader": "6.0.0",
+    "enzyme-adapter-react-16": "1.15.6",
+    "enzyme-to-json": "3.6.2",
+    "eslint": "8.8.0",
+    "eslint-import-resolver-alias": "1.1.2",
+    "eslint-import-resolver-typescript": "2.7.1",
+    "eslint-import-resolver-webpack": "0.13.2",
+    "eslint-plugin-babel": "5.3.1",
+    "eslint-plugin-cypress": "2.12.1",
+    "eslint-plugin-header": "3.1.1",
+    "eslint-plugin-import": "2.25.4",
+    "eslint-plugin-jest": "26.5.3",
+    "eslint-plugin-mattermost": "github:mattermost/eslint-plugin-mattermost",
+    "eslint-plugin-no-only-tests": "2.6.0",
+    "eslint-plugin-react": "7.28.0",
+    "eslint-plugin-react-hooks": "4.3.0",
+    "file-loader": "6.2.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "26.1.0",
-    "jest-canvas-mock": "2.2.0",
-    "jest-junit": "11.0.1",
-    "mattermost-webapp": "github:mattermost/mattermost-webapp#23f5f93d9f12a7e2b5623e5cee6814366abd9a0f",
-    "node-sass": "4.14.1",
-    "sass-loader": "9.0.2",
-    "style-loader": "1.2.1",
-    "webpack": "4.43.0",
-    "webpack-cli": "3.3.12"
+    "isomorphic-fetch": "3.0.0",
+    "jest": "27.4.7",
+    "jest-canvas-mock": "2.3.1",
+    "jest-junit": "13.0.0",
+    "sass": "1.52.3",
+    "sass-loader": "13.0.0",
+    "style-loader": "3.3.1",
+    "webpack": "5.73.0",
+    "webpack-cli": "4.9.2"
   },
   "dependencies": {
-    "core-js": "3.6.5",
-    "mattermost-redux": "5.27.0",
-    "react": "16.13.1",
-    "react-redux": "7.2.0",
-    "redux": "4.0.5",
-    "typescript": "3.9.6"
+    "@mattermost/webapp": "github:mattermost/mattermost-webapp",
```

Comment:
I'm wondering if we can remove this dependency. It's only used for setting up tests, and even then nothing is really Mattermost-specific in that setup.

Reply:
1/5 it might be worth keeping it around as it makes it easier for dev touse it later if the need it.